packages/yoga/src/NavigationMenu/web/Switcher/Switcher.tsx

```diff
@@ -87,11 +89,11 @@ const Switcher = ({
       {Avatar}
 
       <StyledTextContainer>
-        <StyledTitle>
-          {title || <Skeleton type="text" variant="body2" width="100%" />}
-        </StyledTitle>
+        {loadingTitle && <Skeleton type="text" variant="body2" width="100%" />}
```

Comment:
isLoading?

Reply:
The loading should not be specific of the title...

Reply:
> isLoading?

@matheushdsbr is there any rule in place for boolean props? Should this new prop be named `isLoading`? And what about the others already released for this `Navigation Menu` -> `openOnMobile`, `responsive`, `active` and `expanded`?

Reply:
According to the examples already in place, we decide, for now, just add "is" to the loading prop (following the name already used in Button)